libbeat/outputs/elasticsearch/client.go

```diff
@@ -71,6 +71,12 @@ type Connection struct {
 	version string
 }
 
+var connectCallbackRegister connectCallback
```

Comment:
Having this as global means it will also be applied to non-output ES clients (e.g. the one loading the dashboards, or the one loading the pipelines or the one for monitoring). That is perhaps OK, but mentioning it in case someone else thinks of something about that.

Of course, the global is kind of last resort solution. @urso any suggestions on how we can improve this? (Not necessarily in this PR).